README.md

```diff
@@ -95,4 +95,4 @@ chart.show()?;
 - [A Dramatic Tour through Python’s Data Visualization Landscape (including ggplot and Altair) – Regress to Impress](https://dsaber.com/2016/10/02/a-dramatic-tour-through-pythons-data-visualization-landscape-including-ggplot-and-altair/)
 - [Specifying Data in Altair — Altair 3.0.0 documentation](https://altair-viz.github.io/user_guide/data.html#long-form-vs-wide-form-data)
 - [Visualization — list of Rust libraries/crates // Lib.rs](https://lib.rs/visualization)
-- [Quicktype](https://quicktype.io/) (got issue with the alternative https://transform.now.sh/json-to-rust-serde) was used to bootstrap `vegalite.rs` from the [vega-lite's json schema](https://vega.github.io/schema/vega-lite/v3.json)
+- [Quicktype](https://quicktype.io/) (got issue with the alternative https://transform.now.sh/json-to-rust-serde) was used to bootstrap `vegalite.rs` from the [vega-lite's json schema](https://vega.github.io/schema/Vega-Lite/v3.json)
```

Comment:
search and replace too fast, url are case sensitive. this one no longer works (404)

Reply:
Yes search and replace too fast .. I’m sorry ! I’ll fix this